lib/orbit-common/cache.js

```diff
@@ -154,6 +158,23 @@ var Cache = Class.extend({
     if (this.trackRevLinks && (op === 'add' || op === 'replace')) {
       this._addRevLinks(path, value, normalizedOperation);
     }
+
+    return true;
+  },
+
+  _isOperationRequired: function(operation){
+    if(operation.path[2] === '__rel'){
+      var recordExists = this.retrieve([operation.path[0], operation.path[1]]);
+      var valueMatches = this._valuesEqual(this.retrieve(operation.path), operation.value);
+      return recordExists && !valueMatches;
+    }
+
+    return true;
+  },
+
+  _valuesEqual: function(a, b){
+    if(a == null && b == null) return true;
```

Comment:
Is this extra check necessary? I'd prefer not to use `==` for equality testing, and I'm not sure we want to equate `null` with `undefined` regardless.